docs/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+# Related issue(s): 
+- issue goes here
+
+# Description of changes made: 
+- change 1
+- change 2
+
+# (optional) Questions or comments for review:
+- any additional questions or comments goes here. 
```

Comment:
Instead of using top-level headers with `#` let's switch this to sub-headings with `###`.

Reply:
resolving issue '# let's switch this to sub-headings with ### https://github.com/Cosmic-Binturongs/slothstagram/issues/26'